vhdbuilder/packer/install-dependencies.sh

```diff
@@ -225,14 +225,14 @@ if [[ ${installSGX} == "True" ]]; then
         echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
     done
 
-    SGX_PLUGIN_VERSIONS="0.2"
+    SGX_PLUGIN_VERSIONS="0.4"
```

Comment:
you should include both old and new until the AKS (with new image) is released to all regions.

Reply:
Done.